Common/Brokerages/FTXBrokerageModel.cs

```diff
@@ -0,0 +1,109 @@
+        /// </summary>
+        public override IReadOnlyDictionary<SecurityType, string> DefaultMarkets { get; } = GetDefaultMarkets();
+
+
+        /// <summary>
+        /// Gets a new buying power model for the security, returning the default model with the security's configured leverage.
+        /// For cash accounts, leverage = 1 is used.
+        /// For margin trading, max leverage = 5
+        /// </summary>
+        /// <param name="security">The security to get a buying power model for</param>
+        /// <returns>The buying power model for this brokerage/security</returns>
+        public override IBuyingPowerModel GetBuyingPowerModel(Security security)
+        {
+            return AccountType == AccountType.Margin
+                ? new SecurityMarginModel(GetLeverage(security))
+                : new CashBuyingPowerModel();
+        }
+
+        /// <summary>
+        /// Gets the brokerage's leverage for the specified security
+        /// </summary>
+        /// <param name="security">The security's whose leverage we seek</param>
+        /// <returns>The leverage for the specified security</returns>
+        public override decimal GetLeverage(Security security)
+        {
+            if (AccountType == AccountType.Cash)
+            {
+                return 1m;
+            }
+
+            return _defaultLeverage;
+        }
+
+        /// <summary>
+        /// Provides FTX fee model
+        /// </summary>
+        /// <param name="security">The security to get a fee model for</param>
+        /// <returns>The new fee model for this brokerage</returns>
+        public override IFeeModel GetFeeModel(Security security)
+            => new FTXFeeModel();
+
+        private static IReadOnlyDictionary<SecurityType, string> GetDefaultMarkets()
+        {
+            var map = DefaultMarketMap.ToDictionary();
+            map[SecurityType.Crypto] = Market.FTX;
+            return map.ToReadOnlyDictionary();
+        }
+
+        /// <summary>
+        /// Please note that the order's queue priority will be reset, and the order ID of the modified order will be different from that of the original order.
+        /// Also note: this is implemented as cancelling and replacing your order.
+        /// There's a chance that the order meant to be cancelled gets filled and its replacement still gets placed.
+        /// </summary>
+        /// <param name="security"></param>
+        /// <param name="order"></param>
+        /// <param name="request"></param>
+        /// <param name="message"></param>
+        /// <returns></returns>
+        public override bool CanUpdateOrder(Security security, Order order, UpdateOrderRequest request, out BrokerageMessageEvent message)
+        {
+            message =
+                new BrokerageMessageEvent(
+                    BrokerageMessageType.Warning,
+                    0,
+                    "You must cancel and re-create instead.");
+            return false;
+        }
+    }
```

Comment:
Missing to override the `GetBenchmark` check out the other crypto brokerage models 👍